coordinator/txmanager.go

```diff
@@ -161,6 +161,11 @@ func (t *TxManager) NewAuth(ctx context.Context, batchInfo *BatchInfo) (*bind.Tr
 			return nil, tracerr.Wrap(er)
 		}
 	}
+	//If gas price is higher than 2000, probably we are going to get the gasLimit exceed error
+	const masGasPrice = 2000
```

Comment:
I guess there is a typo, the const name would be maxGasPrice. Is that correct?

Reply:
Also, as we already know that maxGasPrice constant will be compared to the gasPrice ( a big.Int value), what if we already define the constant as big.Int? This way it will save several cast later